src/VolleyManagement.Backend/VolleyManagement.UI/Global.asax.cs

```diff
@@ -16,7 +16,9 @@
     using VolleyManagement.UI.Infrastructure.IOC;
 
 #pragma warning disable SA1649 // File name must match first type name
+#pragma warning disable S1200 // Classes should not be coupled to too many other classes (Single Responsibility Principle)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)